README.md

```diff
@@ -54,3 +55,51 @@ This repository implments tests using [pytest](https://docs.pytest.org/), the te
 
 [^env_vars]: [For linux](https://www.gnu.org/software/bash/manual/bash.html#Environment) you can either prepend as in `NAME=value pipenv ...` or use `export NAME=value`, [for windows](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/set_1) you can use `set`.
 [^user_config]: A more detailed documentation on [the user configuration file can be found here](docs/user_configuration_file.md).
+
+## AWS Packing
+Since some services from infin-neat-core code runs at AWS cloud services, it's necessary to properly pack the code to upload it using the web interface, AWS CLI, or AWS CDK. 
+
+For that, on specific script has been created and can be run using pipenv: 
+```
+pipenv run aws_pack
+```
+### Requirements:
+
+#### **Install Docker**: 
+Since in the packing processing Docker is used, it's necessary to install it. For installing Docker, the official documentation from Docker can be followed for [Linux](https://docs.docker.com/desktop/install/linux-install/), [Windows](https://docs.docker.com/desktop/install/windows-install/), and [MAC](https://docs.docker.com/desktop/install/mac-install/), 
+
+
+#### **Running Docker Rootless**: 
```

Comment:
I would delegate instructions to the official links rather than copying the instructions here. We can keep your brief explanation of why we need to configure "rootless" access to Docker, that's context to our specific repository, but specific steps regarding how to setup rootless access to docker should be a link to the official doc.

Two good reasons come to me when writing this suggestion:
1. Keep our documentation shorter and cleaner
2. These instructions might change in the future in the official doc, we don't want the extra effort to keep syncing them.

I'd also add that these instructions are linux only (many distros): https://docs.docker.com/engine/security/rootless/

We can also include that MacOS doesn't need these steps, it has a linux vm in which root access is by default: https://docs.docker.com/desktop/mac/permission-requirements/#containers-running-as-root-within-the-linux-vm